Formula/ldns.rb

```diff
@@ -25,12 +25,12 @@ class Ldns < Formula
 
   depends_on "swig" => :build
   depends_on "openssl@1.1"
-  depends_on "python@3.10"
+  depends_on "python@3.11"
 
   conflicts_with "drill", because: "both install a `drill` binary"
 
   def install
-    python3 = "python3.10"
+    python3 = "python3.11"
     args = %W[
       --prefix=#{prefix}
```

Comment:
Can you add this too while we're at it?
```suggestion
    args = *std_configure_args + %W[
```

Reply:
Thanks @SMillerDev for the proposal. This has been merged upstream at: https://github.com/Homebrew/homebrew-core/commit/4c1f7d3454c8286786f814270111fc7db17706be